Meta/qemu-paths.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+set -e
+
+# sourcing this file will export SERENITY_QEMU_IMG_BIN and SERENITY_QEMU_BIN
+# in such a way that they will contain the correct paths also under WSL
+
+if command -v wslpath >/dev/null; then
+    # existence of wslpath implies we are running inside WSL on Windows
+    # the Windows installation of QEMU adds the install directory of itself
+    # to the registry, we use powershell to get the install directory
+
+    # Use full path to powershell, just in case it is not on the path already
+    QEMU_INSTALL_DIR=$(/mnt/c/Windows/System32/WindowsPowershell/v1.0/powershell.exe -Command Get-ItemPropertyValue 'HKLM:\SOFTWARE\QEMU' -Name Install_Dir)
```

Comment:
If a new version of Windows releases with v1.1 or v2.0 or ... a different version of powershell, will this script break?

Reply:
Considering that the current version off PowerShell is already 2.0 even though it is located on that path, I doubt it will break.

Reply:
On the other hand, we require That system32 and cohorts are in the WSL path, so you should be able to elide the path and just use `PowerShell.exe`